astropy/modeling/functional_models.py

```diff
@@ -854,6 +854,22 @@ def fit_deriv(x, amplitude, x_0, fwhm):
         d_fwhm = 2 * amplitude * d_amplitude / fwhm * (1 - d_amplitude)
         return [d_amplitude, d_x_0, d_fwhm]
 
+    def bounding_box(self, factor=25):
+        """Tuple defining the default ``bounding_box`` limits,
+        ``(x_low, x_high)``.
+
+        Parameters
+        ----------
+        factor : float
+            The multiple of FWHM used to define the limits.
+            Similar to `Gaussian1D`.
```

Comment:
I'll remove that line then. The factor is chosen such that 99% of the "flux" is contained within the bounding box (this is tested as part of the example model thingy you see in this PR). I don't want the bounding box to include the very extended part of the wing, because then you cannot really see the feature of interest when you plot the model against bounding box limits. Does this make sense?